src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferenceCache.cs

```diff
@@ -108,7 +108,7 @@ public static SymbolInfo GetSymbolInfo(SemanticModel model, SyntaxNode node, Can
             }
 
             static bool IsCandidate(SyntaxToken t, ISyntaxFactsService syntaxFacts, string text)
-                => syntaxFacts.IsGlobalNamespaceKeyword(t) || (syntaxFacts.IsIdentifier(t) && syntaxFacts.TextMatch(t.ValueText, text));
+                => syntaxFacts.IsIdentifier(t) && syntaxFacts.TextMatch(t.ValueText, text);
```

Comment:
ugly hack where we used the same path for identifier lookup to lookup the 'Global' keyword.